assets/index.tmpl

```diff
@@ -3,7 +3,14 @@
 <div class="maincontents">
 
 {{ if .ProductUrl -}}
-<h1>{{ .ProductName }} Documentation Server</h1>
+<h1>
+  {{ if .LogoUrl -}}
+  <img src="{{ .LogoUrl }}" alt="Logo" width="16" height="16">
+  {{ else -}}
+  {{ template "icon" . }}
+  {{ end -}}
+  {{ .ProductName }} Documentation Server
+</h1>
 {{ else -}}
```

Comment:
This looks wrong: We check if a ProductUrl exists, but are not using it. Should be most likely a check for ProductName?
And I wouldn't use the fallback icon here, if no Logo for the product exists, we shouldn't display a "book", instead nothing. If a product does not have a logo, don't show one.

Reply:
> This looks wrong: We check if a ProductUrl exists, but are not using it. Should be most likely a check for ProductName? 

Yeah, you are right, I haven't looked at it, since I didn't have a need to modify it.

> And I wouldn't use the fallback icon here, if no Logo for the product exists, we shouldn't display a "book", instead nothing. If a product does not have a logo, don't show one.

Good point